Lib/test/test_xml_etree.py

```diff
@@ -1468,6 +1464,7 @@ def _feed(self, parser, data, chunk_size=None):
         else:
             for i in range(0, len(data), chunk_size):
                 parser.feed(data[i:i+chunk_size])
+        parser.flush()
```

Comment:
Some tests are passed without it.
```suggestion
            if chunk_size < 21:
                parser.flush()
```

Reply:
@serhiy-storchaka okay but why be magic, cryptic, brittle here — what's the gain?

Reply:
`flush()` is not necessary. If always use `flush()` in tests we can miss a regression when such code will stop working without `flush()`. If you don't like magic constant 21 here, add parameter `flush=False` and call this function with `flush=True` in test_simple_xml_chunk_1 and test_simple_xml_chunk_5.

Reply:
TLDR: Implemented and pushed.

@serhiy-storchaka adding `flush=False` will be quite noisy and complex since it needs (a) propagation through `self._feed` for all calls from `test_simple_xml` or (b) it needs new instance state a la `self._flush = False`. But it seems needed if you care as much as avoiding unnecessary flushes… because `chunk_size < 21` would only happen to magically work, there is no match to anything >=21 within Expat, and we may push smaller chunks than that with `self._feed`.